src/ui/server/constants/routes/insurance/index.ts

```diff
@@ -45,7 +45,6 @@ export const INSURANCE_ROUTES = {
     NEED_TO_START_AGAIN: `${INSURANCE_ROOT}${ELIGIBILITY_ROOT}/need-to-start-again`,
     CONTRACT_TOO_SHORT: `${INSURANCE_ROOT}${ELIGIBILITY_ROOT}/contract-too-short`,
   },
-  APPLY_OFFLINE: `${INSURANCE_ROOT}/apply-using-our-form`,
```

Comment:
Still some instances of `APPLY_OFFLINE` in template constants and content strings, can these be removed now?

Reply:
I'll address this after merge